vendor/engines/split_accounts/spec/services/order_detail_splitter_spec.rb

```diff
@@ -173,7 +173,9 @@
 
     it "does not do anything to the reservation" do
       order_detail.update_attributes!(account: split_account)
-      described_class.new(order_detail, split_time_data: true).split
+      expect {
```

Comment:
This is not directly related to this fix. It's an old issue where splitting order details was sometimes deleting reservations. This is just a little more explicit in what it's testing than the original test.